### DISCUSSION TITLE
Future of the dropwizard.io domain?

### DISCUSSION THREAD

**Discussion by jplock** [dropwizard/dropwizard#9412] [General]
https://every.to/p/the-disappearance-of-an-internet-domain 🤔 

I've been renewing and paying for this in my personal AWS account.

**Comment by joschi** [comment]
> The IANA may fudge its own rules and allow .io to continue to exist. Money talks, and there is a lot of it tied up in .io domains. However, the history of the USSR and Yugoslavia still looms large, and the IANA may feel that playing fast and loose with top-level domains will only come back to haunt it.

I doubt that the `.io` TLD will really vanish but probably some commercial entity will step in and take over stewardship.

In other words, I'd sit that one out, unless you want to change the ownership/maintenance of the dropwizard.io domain in general.

And before I forget: Thank you so much for carrying the dropwizard.io domain for so long for the project! ❤️

**Reply by jplock** [reply]
Yea I’m skeptical anything will happen but interesting to see how it plays out
